hack/make.sh

```diff
@@ -16,26 +20,64 @@ if [[ $SOURCEDIR != $DIR ]];then
 	ln -sf $DIR/ $SOURCEDIR
 fi
 
-# install pouch and essential binaries: containerd, runc and so on
-function install_pouch ()
+function get_containerd_version()
 {
-	# install containerd
-	echo "Download and install containerd."
-	wget --quiet https://github.com/containerd/containerd/releases/download/v1.0.3/containerd-1.0.3.linux-amd64.tar.gz -P $TMP
-	tar xf $TMP/containerd-1.0.3.linux-amd64.tar.gz -C $TMP && cp -f $TMP/bin/* /usr/local/bin/
+	if which containerd &>/dev/null; then
+		CONTAINERD_VERSION=$(containerd -v|cut -d " " -f 3)
+	fi
+}
 
-	# install runc
-	echo "Download and install runc."
-	wget --quiet https://github.com/alibaba/runc/releases/download/v1.0.0-rc4-1/runc.amd64 -P /usr/local/bin
-	chmod +x /usr/local/bin/runc.amd64
-	mv /usr/local/bin/runc.amd64 /usr/local/bin/runc
+function get_runc_version()
+{
+	if which runc &>/dev/null; then
+		RUNC_VERSION=$(runc -v|head -1| cut -d " " -f 3)
+	fi
+}
 
-	# copy pouch daemon and pouch cli to PATH
-	echo "Install pouch."
-	cp -f $DIR/pouch $DIR/pouchd /usr/local/bin/
-	
-	# install lxcfs
-	echo "Install lxcfs"
+function get_nsenter_version()
+{
+	if which nsenter &>/dev/null; then
+		NSENTER_VERSION=$(nsenter -V | cut -d " " -f 4)
+	fi	
+}
+
+function get_dumb_init_version()
+{
+	if which dumb-init &>/dev/null; then
+		DUMB_INIT_VERSION=$(dumb-init -V 2>&1 | cut -d " " -f 2)
+	fi	
+}
+
+function install_containerd()
+{
+	echo "Try installing containerd"
+	get_containerd_version
+	if [ "$CONTAINERD_VERSION" = "v1.0.3" ]; then
```

Comment:
`=` to be `==`
And I would prefer user `[[ ]]`, same comments for the following code.